sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,370 @@
+(@OGUID+92, 202752, 732, 1, 1, 0, -1139.731, 1304.658, 120.4041, 0, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+93, 202752, 732, 1, 1, 0, -1188.135, 1229.25, 119.3753, 5.67232, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 5399)
+(@OGUID+94, 202752, 732, 1, 1, 0, -1207.754, 1193.16, 119.875, 5.67232, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+95, 202752, 732, 1, 1, 0, -1218.727, 1295.231, 119.3646, 5.270896, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+96, 202752, 732, 1, 1, 0, -1228.863, 1326.564, 126.1719, 5.270896, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+97, 202752, 732, 1, 1, 0, -1232.319, 1140.254, 120.0816, 5.67232, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+98, 202752, 732, 1, 1, 0, -1251.938, 1104.16, 120.9601, 5.67232, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 5399)
+(@OGUID+99, 202752, 732, 1, 1, 0, -1267.78, 1199.09, 118.9367, 5.811947, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 5399)
+(@OGUID+100, 202752, 732, 1, 1, 0, -1274.689, 1289.168, 119.1615, 5.270896, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+101, 202752, 732, 1, 1, 0, -1291.691, 1221.736, 119.4794, 5.811947, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+(@OGUID+102, 202752, 732, 1, 1, 0, -1313.983, 1166.955, 119.0399, 5.811947, 0, 0, 0, 1, 120, 255, 1), -- 202752 (Area: 0)
+-- Mining: Rich Elementium Vein, Rich Pyrite Deposit, Elementium Vein (12/22 spawns) 
+(@OGUID+103, 202738, 732, 1, 1, 0, -535.4896, 1142.139, 89.60479, 0, 0, 0, 0, 1, 120, 255, 1), -- 202738 (Area: 0)
+(@OGUID+104, 202738, 732, 1, 1, 0, -541.8698, 1354.247, 24.02083, 0, 0, 0, 0, 1, 120, 255, 1), -- 202738 (Area: 5546)
+(@OGUID+105, 202741, 732, 1, 1, 0, -953.9479, 1136.99, 125.4949, 0, 0, 0, 0, 1, 120, 255, 1), -- 43.3 29.1
+(@OGUID+106, 202740, 732, 1, 1, 0, -956.7795, 1279.313, 131.099, 0, 0, 0, 0, 1, 120, 255, 1), -- 202740 (Area: 0)
+(@OGUID+107, 202738, 732, 1, 1, 0, -1002.42, 1138.372, 124.6493, 5.009095, 0, 0, 0, 1, 120, 255, 1), -- 202738 (Area: 0)
+(@OGUID+108, 202738, 732, 1, 1, 0, -1032.792, 1078.299, 125.4842, 0, 0, 0, 0, 1, 120, 255, 1), -- 202738 (Area: 0)
+(@OGUID+109, 202738, 732, 1, 1, 0, -1234.997, 1431.396, 126.5013, 0, 0, 0, 0, 1, 120, 255, 1), -- 202738 (Area: 0)
+(@OGUID+110, 202740, 732, 1, 1, 0, -1291.189, 1370.056, 130.3137, 0, 0, 0, 0, 1, 120, 255, 1), -- 202740 (Area: 0)
+(@OGUID+111, 202738, 732, 1, 1, 0, -1341.788, 1337.564, 134.5677, 5.724681, 0, 0, 0, 1, 120, 255, 1), -- 33.4 58
+(@OGUID+112, 202740, 732, 1, 1, 0, -1357.306, 1088.785, 123.8459, 0, 0, 0, 0, 1, 120, 255, 1), -- 202740 (Area: 5399)
+(@OGUID+113, 202741, 732, 1, 1, 0, -1368.493, 603.5156, 127.235, 0, 0, 0, 0, 1, 120, 255, 1), -- 69.7 60.1
+(@OGUID+114, 202741, 732, 1, 1, 0, -1425.139, 1038.675, 121.9771, 0, 0, 0, 0, 1, 120, 255, 1); -- 202741 (Area: 5399)
+*/
+
+-- Corrections
+UPDATE `gameobject_template` SET `Data4` = 26204, `Data5` = 26205 WHERE `entry` = 205103;
```

Comment:
https://i.imgsafe.org/0c93361.png
Hacky, but I have no sniffs for it, don't know what else to do.